test/passing/extensions.ml.ref

```diff
@@ -81,3 +81,21 @@ let _ =
   [%expr
     let x = e in
     f y] [@x]
+
+let _ =
+  f
+    ( for i = 0 to 1 do
+        ()
+      done )
+    ( while true do
+        ()
+      done )
+
+let _ =
+  f
+    ( for%ext i = 0 to 1 do
+        ()
+      done )
+    ( while%ext true do
+        ()
+      done )
```

Comment:
:-) where do you find such code...

Reply:
I think it was something like
```ocaml
dont_wait_for begin
   while%lwt cond do
     expr
   done
end
```

I've started to run ocamlformat against all packages available in the opam-repository